project/plugins.sbt

```diff
@@ -4,6 +4,6 @@
 
 addSbtPlugin("org.scalameta"  % "sbt-scalafmt"       % "2.5.4")
 addSbtPlugin("com.eed3si9n"   % "sbt-buildinfo"      % "0.13.1")
-addSbtPlugin("com.github.sbt" % "sbt-ci-release"     % "1.9.3")
+addSbtPlugin("com.github.sbt" % "sbt-ci-release"     % "1.11.0")
```

Comment:
I'm in the process of migrating com.github.sbt to the Central Portal. Please update to sbt-ci-release 1.11.0 only after that's done.